bridge-sdk/connectors/omni-connector/src/omni_connector.rs

```diff
@@ -325,7 +365,7 @@ impl OmniConnector {
         }
 
         // TODO: Provide fee and nativeFee
-        let withdraw_call = factory.init_transfer(near_token_id, amount, 0, 0, receiver);
+        let withdraw_call = factory.init_transfer(erc20_address, amount, 0, 0, receiver, String::new());
```

Comment:
Let's add a proper fee in this PR. It should take too much time. What do you think?

Reply:
Done